postgrest.cabal

```diff
@@ -24,6 +24,7 @@ Flag CI
 executable postgrest
   main-is:             PostgREST/Main.hs
   default-extensions:  OverloadedStrings, ScopedTypeVariables, QuasiQuotes, LambdaCase
+  ghc-options:         -threaded -rtsopts -with-rtsopts=-N
```

Comment:
-O2 used to be in cabal before (all of them were there), only recently you moved them to stack